locales/en-US/experiments.ftl

```diff
@@ -159,6 +159,18 @@ pageshotDetails2Copy = Find saved screenshots easily. Browse thumbnails in grid
 pageshotContributors0TitleEngineer = Software Engineer
 pageshotContributors1Title = Software Engineer
 pageshotContributors2Title = UX Designer
+pricescoutDescription = Aliquam quis orci id ex malesuada congue. Nullam fringilla nulla rutrum, placerat elit et, sodales urna.
+pricescoutDetails0Copy = Maecenas rutrum sapien diam, vestibulum posuere odio ornare pellentesque.
+pricescoutDetails1Copy = Donec aliquet aliquet augue sed viverra.
+pricescoutDetails2Copy = Sed viverra ligula ex, ut pellentesque mauris maximus id.
+pricescoutIntroduction = Nunc id orci eget erat suscipit sollicitudin eleifend at massa. Sed vestibulum est vel vulputate hendrerit. Duis purus mauris, feugiat et justo a, molestie tincidunt nulla. Donec sodales neque sit amet enim ornare ullamcorper.
+pricescoutLegalnotice = By proceeding, you agree to the <terms-link>terms</terms-link> and <privacy-link>privacy</privacy-link> policies of Test Pilot and the <modal-link>Price Scout privacy policy</modal-link>.
+pricescoutMeasurements0 = Product Pages: Price Scout collects how often you visit saved product pages.
+pricescoutMeasurements1 = Product Data: Price Scout collects data about the products you choose to track such as prices change frequency, size, and the accuracy of that information.
+pricescoutMeasurements2 = Interaction Data: Price Scout collects information about how you use the feature such as how often you add, delete and interact with saved products, number of products you choose to track, and how you engage with messages and push notifications.
+pricescoutToursteps0Copy = Integer augue turpis.
+pricescoutToursteps1Copy = Donec quis leo odio.
+pricescoutToursteps2Copy = In gravida urna ut ante sollicitudin.
```

Comment:
@chuckharmston Note that a bunch of these keys (and some strings) still refer to "scout", as well as a couple files:

```
BIN +879 KB frontend/src/images/experiments/price-wise/social/price-scout-facebook.png
BIN +191 KB frontend/src/images/experiments/price-wise/social/price-scout-twitter.png
```

Looks like the .yaml shows them as price-wise-*.png, but the assets themselves need to be renamed to match the YAML values:

```yaml
image_facebook: /static/images/experiments/price-wise/social/price-wise-facebook.png
image_twitter: /static/images/experiments/price-wise/social/price-wise-twitter.png
```